src/app/(dashboard)/page.tsx

```
@@ -2,15 +2,20 @@ import { headers } from "next/headers";
+  const data = await caller.hello({ text: "Andrew Server" });
+
   const session = await auth.api.getSession({
     headers: await headers(),
   });
   if (!session) {
     redirect("/sign-in");
   }
 
+  return <p>{data.greeting}</p>;
+
   return <HomeView />;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)